app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -288,7 +288,6 @@ class StateFragmentTest {
   }
 
   @Test
-  @DisableAccessibilityChecks // TODO(#3251): Enable AccessibilityChecks
```

Comment:
The changes in toolbar and content/feedback item fixes `36` test cases in this. `10` test cases are still pending which will be fixed later.

Reply:
<img width="1478" alt="Screenshot 2021-06-24 at 3 16 58 AM" src="https://user-images.githubusercontent.com/9396084/123172353-a8625200-d49a-11eb-80e9-2da13814304d.png">